src/cunumeric/execution_policy/execution_policy_helpers.h

```diff
@@ -0,0 +1,11 @@
+#ifndef SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
+#define SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
+
+// If nvcc or clang indicates CUDA compilation
+#if defined(__CUDACC__) || defined(__CUDA__)
+#define CUDA_FUNCTION __host__ __device__
+#else
+#define CUDA_FUNCTION
+#endif
+
+#endif  // SRC_CUNUMERIC_EXECUTION_POLICY_EXECUTION_POLICY_HELPERS_H_
```

Comment:
Missing newline at end of file. Weird that the clang auto-formatter didn't catch that. Does it look like clang-format is running when you make change to header files? It might be that the filename pattern is skipping header files, or maybe just our clang-format configuration isn't checking for missing newlines at end of files.